app/models/application.rb

```diff
@@ -0,0 +1,9 @@
+class Application < ApplicationRecord
+  validates :phone_number, :street, :city, :state, :zip_code, :country, :birth_place, :birth_date,           :country_of_citizenship, :occupation, presence: true
```

Comment:
Lets try to keep the length of code under 80 or so. Split it under multiple lines for readability. (Probably a tab issue or something)